presets/4.5/rates/rubberquads_team_rates.txt

```diff
@@ -0,0 +1,85 @@
+#$ TITLE: RubberQuads Team Rates
+#$ FIRMWARE_VERSION: 4.3
+#$ FIRMWARE_VERSION: 4.4
+#$ FIRMWARE_VERSION: 4.5
+#$ CATEGORY: RATES
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: rates, freestyle, race, der_V, settek, rubberquads
+#$ AUTHOR: Volker Göschl-Covrljan
+
+#$ PARSER: MARKED
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: <img src="https://www.rubberquads.com/static/rubberquads/RQ-logo_header.png" width="200px" style="margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: #Rates from Team RubberQuads
+#$ DESCRIPTION: This is what we like to rock on our sticks.
+#$ DESCRIPTION: I (Volker) am a thumber so my center rates are pretty low to give me the ability to fly smooth and precise lines 
+#$ DESCRIPTION: My race-rates are toned down at the edge (550dps) but the same at the center to allow for a seamless transition 
+#$ DESCRIPTION: between ze flippyfloppies and racing/cinematic
+#$ DESCRIPTION: Settek is flying hybrid so his rates are quite a bit higher around center-stick
```

Comment:
just noticed that this rate changes also the throttle curve, worth mentioning in the description

Reply:
totally!